Mobile/ShopLens/Droid/ShopLens.Droid.csproj.user

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <SelectedDevice>Android_Accelerated_x86_Oreo</SelectedDevice>
+    <SelectedDevice>Motorola Moto G %285%29</SelectedDevice>
```

Comment:
This file probably shouldn't be commited?

Reply:
Yeah, I think so, I'll add a gitignore statement for it.